content/teams/Code Blooded/Index.md

```diff
@@ -0,0 +1,125 @@
+---
+type: "team"                                                        
+title: Code Blooded
+name: "Code Blooded"
+github-link: "https://github.com/KaranBengani "
+description: "While("CODE BLOODED"=="BEST TEAM")
+               { printf("we will win");}"
```

Comment:
@KaranBengani , Have a look at your Description : DO not include any sort of special symbols write in words..

The way you should go with:

Description: "Here Goes your Teams description"

Reply:
One line of description is sufficient.

Reply:
i have updated it, i hope its fine now